src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { FC } from 'react';
+import { User } from '../../types';
+
+type Props = {
+  user?: User;
+};
+
+export const UserInfo: FC<Props> = ({ user }) => {
+  return (
+    user && (
+      <a className="UserInfo" href={`mailto:${user.email}`}>
+        {user.name}
+      </a>
```

Comment:
Don't render the component if the property that you pass to the component has null or undefined value. [[checklist]](https://github.com/mate-academy/react_add-todo-form/blob/master/checklist.md)